components/dashboard/dashboard.go

```diff
@@ -91,11 +92,18 @@ func (d *Dashboard) ReconcileComponent(owner metav1.Object, cli client.Client, s
 		if err != nil {
 			return fmt.Errorf("failed to set dashboard ISV from %s", PathISVAddOn)
 		}
+		// ConsoleLink handling
+		err = deploy.DeployManifestsFromPath(owner, cli, ComponentName,
```

Comment:
Wouldn't we want to get url from dashboard route and update the ConsoleLink yaml first https://github.com/red-hat-data-services/odh-manifests/blob/feature-rearchitecture/odh-dashboard/consolelink/consolelink.yaml#L9 before applying it ?

Reply:
strange, the other file of generating replacement is not in this PR. must get lost in the old branch. 
i will update it

Reply:
not sure if this is the right way.
so i want to reuse ReplaceStringsInFile function to replace variable in manifests, and it is currently in dscinitialzation.utils, to avoid cycle import, i moved the function into pkg/common
in theory, these lines of get route, replace can be done in a function instead of inject in the function

Reply:
Moved the RelaceStrings to common, is this ready to merge ?

Reply:
yes, think so.
it is using `common.ReplaceStringsInFile()`